terraform_deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Minor, but worth getting into the habit of doing
```suggestion
#!/bin/bash -e
```
or a separate `set -e` so that any line failing causes the script to exit.